librad/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "librad"
 version = "0.1.0"
-authors = ["The radicle Team <dev@radicle.xyz>"]
+authors = ["The Radicle Team <dev@radicle.xyz>"]
```

Comment:
Isn't it _supposed_ to be lowercase?

Reply:
No, I checked with ingrrish and brand rranguage native :)

Reply:
I thought the branding people said it's lowercase :thinking: